wdl-analysis/src/analyzer.rs

```diff
@@ -27,7 +27,6 @@ use tokio::sync::mpsc;
 use tokio::sync::oneshot;
 use url::Url;
 use walkdir::WalkDir;
-use wdl_ast::AstNode;
```

Comment:
We're still calling `Document::cast` which is a member of this trait; I don't quite know how CI would be green with this change.

Reply:
Oh never mind; we've modified the trait as part of these changes.